src/fontinfo.rs

```diff
@@ -0,0 +1,940 @@
+use serde::de::Deserializer;
+use serde::ser::{SerializeSeq, Serializer};
+use serde::{Deserialize, Serialize};
+
+use crate::shared_types::Guideline;
+use crate::Error;
+
+// The specification is vague about data type limits, usually implicitly meaning
+// Python types. Since Python is dynamic, the spec does not nail down the exact type
+// in several locations and we have to assume the biggest type that can hold it.
+type Integer = i32;
```

Comment:
is it worth making these 64 bit types? my gut says no, but it's strictly 'more' correct. (being fully correct would probably require some fancy real number type with arbitrary precision)

Reply:
I stayed with 32 bits because sane values should stay inside the possible range.